src/main/java/org/jitsi_modified/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimation.java

```diff
@@ -683,7 +662,7 @@ private synchronized void fireBandwidthEstimationChanged(
      * This class records statistics information about how much time we spend
      * in different loss-states (loss-free, loss-limited and loss-degraded).
      */
-    public class StatisticsImpl implements Statistics
+    public class StatisticsImpl
```

Comment:
do we no longer need to conform to the `Statistics` interface?  If not, maybe this class should be renamed?